### DISCUSSION TITLE
how to create table for this json through serverpod

### DISCUSSION THREAD

**Discussion by Shubhamdsingh** [serverpod/serverpod#484] [Q&A] [answered]
![Screenshot 2022-12-13 at 1 04 55 PM](https://user-images.githubusercontent.com/118890814/207254218-97f3a342-051e-4f63-b265-b4f0b181f758.png)

@yahu1031

**Comment by yahu1031** [comment]
Try this to convert a json to yaml https://www.json2yaml.com/.

**Comment by yahu1031** [comment]
@Shubhamdsingh Please create discussions if you have questions. If you find any issues, create a ticket here.

**Comment by vlidholt** [comment, marked as answer]
If you need to return very specific JSON code, this can be done through Serverpod's web server Relic. It's still experimental, but you can find some information about it here: https://docs.serverpod.dev/concepts/webserver

There is a `WidgetJson` that you can use to generate custom JSON responses.

However, generally, you will want to use the built in Serverpod serialization to pass your objects between the server and your app. Then there is no need for status or statusCode fields, etc.
